sys/ztimer/Makefile.dep

```diff
@@ -23,11 +23,11 @@ ifneq (,$(filter ztimer,$(USEMODULE)))
   endif
 endif
 
-# unless ztimer_xtimer_compat is used, make xtimer use ztimer as backend.
-ifneq (,$(filter ztimer,$(USEMODULE)))
+# unless ztimer_xtimer_compat is used, make xtimer use ztimer_usec as backend.
+ifneq (,$(filter ztimer_periph_timer,$(USEMODULE)))
```

Comment:
I'm not familliar with the new ztimer module, but would `ztimer%` make sense here @kaspar030 ?

Reply:
IMHO, that ist too wide. I came up with this PR when I tried to multiplex `periph_rtt` and, thus, just required `ztimer_msec`. This works perfectly and doesn't interfere with `xtimer` as `ztimer` doesn't make use of `periph_timer`.

Without this PR `xtimer_on_ztimer` is pulled in anyway and creates unnecessary complexity.

Reply:
I think this is alright. I intentionally tried to minimize xtimer code (drop xtimers extension code) when ztimer is used, but that behavior can still be had by also using ztimer_msec.

Reply:
> I think this is alright.

So that's an ACK? :wink:

Reply:
Good question. ;) I'm thinking if it would make sense to make this not dependent on ztimer_periph, but ztimer_msec. but then it might clash when ztimer is not using msec but the same periph that xtimer wants to use.

so yes :)